src/controllers/manual-address.js

```diff
@@ -1,8 +1,75 @@
 import {ReturnState} from './_base.js';
+import utils from 'naturescot-utils';
 
-const manualAddressController = () => {
-  // Much like the start page, the only way out of the before you start page is onwards,
-  // so return success and continue the form.
+/**
+ * Clean the incoming POST request body to make it more compatible with the
+ * database and its validation rules.
+ *
+ * @param {object} body The incoming request's body.
+ * @returns {any} A json object that's just got our cleaned up fields on it.
+ */
+const cleanInput = (body) => {
+  return {
+    // The strings are trimmed for leading and trailing whitespace and then
+    // copied across if they're in the POST body or are set to undefined if
+    // they're missing.
+    addressLine1: body.addressLine1 === undefined ? undefined : body.addressLine1.trim(),
+    addressLine2: body.addressLine2 === undefined ? undefined : body.addressLine2.trim(),
+    addressTown: body.addressTown === undefined ? undefined : body.addressTown.trim(),
+    addressCounty: body.addressCounty === undefined ? undefined : body.addressCounty.trim(),
+    addressPostcode: body.addressPostcode === undefined ? undefined : body.addressPostcode.trim()
+  };
+};
+
+const manualAddressController = (request) => {
+  // Clear errors.
+  request.session.addressError = false;
+  request.session.addressLine1Error = false;
+  request.session.addressTownError = false;
+  request.session.addressCountyError = false;
+  request.session.postcodeError = false;
+  request.session.invalidPostcodeError = false;
+  console.log(request.body);
+  // Clean up the user's input before we store it in the session.
+  const cleanForm = cleanInput(request.body);
+  console.log(cleanForm);
```

Comment:
And this. You can get the same info from running the application through the debugger :)